mypy/server/astdiff.py

```diff
@@ -193,6 +193,8 @@ def snapshot_definition(node: Optional[SymbolNode],
                  node.fallback_to_any,
                  node.is_named_tuple,
                  node.is_newtype,
+                 # Whether subclasses are abstract can depend on the set of abstract attrs
+                 tuple(sorted(node.abstract_attributes)),
```

Comment:
This change can make changes to abstract classes much more expensive to process that normal classes. I have an idea about how to do this without having the trigger the whole class. Let's not merge this for now.